Source/JavaScriptCore/runtime/JSGenericTypedArrayViewConstructor.cpp

```diff
@@ -87,18 +87,18 @@ JSC_DEFINE_HOST_FUNCTION(uint8ArrayConstructorFromBase64, (JSGlobalObject* globa
     StringView view = jsString->view(globalObject);
     RETURN_IF_EXCEPTION(scope, { });
 
-    auto [shouldThrowError, readLength, writeData] = fromBase64(view, std::numeric_limits<size_t>::max(), alphabet, lastChunkHandling);
-    if (shouldThrowError == FromBase64ShouldThrowError::Yes)
+    Vector<uint8_t, 128> output;
```

Comment:
Why don't we just do `Vector<uint8_t, maxLengthFromBase64(view)> output;` here?

Reply:
`maxLengthFromBase64` depends on length, so it is not constant size.

Reply:
Oops, right.